integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/CreatePreAggregateTableCommand.scala

```diff
@@ -61,7 +66,7 @@ case class CreatePreAggregateTableCommand(
     val dbName = cm.databaseName
     LOGGER.audit(s"Creating Table with Database name [$dbName] and Table name [$tbName]")
     // getting the parent table
-    val parentTable = PreAggregateUtil.getParentCarbonTable(dataFrame.logicalPlan)
+    val parentTable = PreAggregateUtil.getParentCarbonTable(logicalPlan)
```

Comment:
most of the content of class same as `CreateTableCommand`, so better call that command from here